### PR TITLE
Controllers api

### PR DESCRIPTION
**Pull request by yevgeniychaplygin3**

### CODE REVIEW

**Review by NathanJesudason — APPROVED** [review]

File: ClientApp/src/app/shared/team-member.model.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export class TeamMember {
```
Comment (hardins01) [inline ClientApp/src/app/shared/team-member.model.ts]: I've been making these types of things in the `ClientApp/types` directory, are you opposed to moving this in there?

File: appsettings.json [hunk 2]
```diff
@@ -1,13 +0,0 @@
-{
```
Comment (hardins01) [inline appsettings.json:1]: Wait, why is appsettings.json being deleted? I swear I just saw you work with it on your machine...

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (hardins01) [inline package-lock.json]: Make sure to delete this extra package-lock.json

**Review by hardins01 — CHANGES_REQUESTED** [review]

File: appsettings.json (hunk 2 above)
Comment (yevgeniychaplygin3) [reply appsettings.json:1]: git ignore was not ignoring it

Comment (hardins01) [reply appsettings.json:1]: We don't want gitignore to ignore it, do we? It has useful stuff in there

File: package-lock.json (hunk 3 above)
Comment (yevgeniychaplygin3) [reply package-lock.json]: forgot to commit the deletion

**Review by hardins01 — APPROVED** [review]

**Pull request merged**